### PR TITLE
doc-4174-os-to-ce

### PR DESCRIPTION
**Pull request by colin-hillis**
- 50-60 instances of this change across multiple files within 4.9.3.

### CODE REVIEW

**Review by paulmoloneyr3 — DISMISSED** [review]

**Review by yarivus — CHANGES_REQUESTED** [review]
The .DS_store file should not be added to the repo it's a local file in Windows OS unrelated to our code.

**Review by AnnaK-R3 — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -663,3 +663,6 @@ repos/**
 **/xml/**
 repos/
 /resources/_gen/
+
+# Desktop Services Store holds meta information about macOS folders after using Finder
+.DS_Store
```
Comment (yarivus) [inline .gitignore:668]: Not enough - as you already ```git add``` for this file ...
So now you need to revert the git addition of this file - along with the .gitignore ;)

Comment (colin-hillis) [reply .gitignore:668]: Added to git-ignore first and then, when I saw result, I committed removal of file.

Comment (colin-hillis) [reply .gitignore:668]: It is a strange little file that auto-generates based on searches in Finder. A real pain...

**Review by yarivus — APPROVED** [review]
LGTM

**Comment by colin-hillis** [comment]
> 

Thanks Yariv.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**